src/platforms/ios/Podfile

```diff
@@ -1,10 +1,10 @@
-pod 'FaceTracker', :git => "git@gitlab.com:cyberlabsai/utils/mobilenative/facetrack-swift.git", :branch => 'feature/refactory'
+pod 'YoonitCamera', :git => "git@github.com:Yoonit-Labs/ios-yoonit-camera.git", :branch => 'development'
```

Comment:
Let use `https://github.com/Yoonit-Labs/ios-yoonit-camera.git` and not `git@github.com:Yoonit-Labs/ios-yoonit-camera.git`.

Because the repository will be public soon.